packages/openfeature-js/src/lib/errors.ts

```diff
@@ -0,0 +1,23 @@
+export type ErrorCodes = 'GENERAL_ERROR' | 'FLAG_TYPE_ERROR' | 'FLAG_VALUE_PARSE_ERROR';
```

Comment:
This may be better as an enum so it can be used at runtime.

Reply:
Yep, done.